broker/docker/docker-compose.yml

```diff
@@ -0,0 +1,5 @@
+version: "2"
+
+services:
+    tunneldigger:
+        build: .
```

Comment:
Does docker-compose really simplify anything here?

(Note: I have basically no experience with docker-compose directly, but suffered a lot of pain from the Discourse docker stuff which I think uses docker-compose.)